src/features/game/components/GameError.tsx

```diff
@@ -1,9 +1,17 @@
-import React from "react";
+import React, { useEffect } from "react";
 import deathAnimation from "assets/npcs/human_death.gif";
 
 export const GameError: React.FC = () => {
+  useEffect(() => {
```

Comment:
Is the purpose of this code to prevent click actions behind the modal?

Is it possible to achieve this using a HTML overlay in this component instead that prevents pointer events behind the modal?

I'm concerned some bugs may pop up if we start transitioning out the `error` state that we are not cleaning up these values

Reply:
What would prevent the user from removing the html overlay from the dom? I would add a clean up in this use effect to remove this? That way when it’s closed pointer events are back in action?